src/core/core.tooltip.js

```diff
@@ -40,6 +40,9 @@ module.exports = function(Chart) {
 		displayColors: true,
 		callbacks: {
 			// Args are: (tooltipItems, data)
+			tooltipOpened: helpers.noop,
```

Comment:
The reason I ask is that these are callbacks and they don't use `Chart.plugins.notify` which notifies all of the plugins